packages/11ty/_includes/components/figure/video/print.js

```diff
@@ -27,11 +36,13 @@ module.exports = function(eleventyConfig) {
       ? poster
       : path.join(imageDir, poster)
     const labelElement = figureLabel({ caption, id, label })
-    const captionElement = figureCaption({ caption, content: labelElement,  credit })
+    const captionElement = figureCaption({ caption, content: labelElement,  credit, mediaId, mediaType })
+
+    const trimLeadingSlash = (string) => string.startsWith('/') ? string.substr(1) : string
 
     return html`
       <div class="q-figure__media-wrapper--${ aspectRatio || 'widescreen' }">
-        <img src="${posterSrc}" />
+        <img src="${trimLeadingSlash(posterSrc)}" />
```

Comment:
this should `trimLeadingSlashed` poster url should theoretically fix the missing poster images in videos in print output, but those are still rendering a broken link due to not being copied into `_site` (due to vite perhaps?) ... something to fix in another PR